src/sidebarLearn.json

```
@@ -150,7 +150,7 @@
           "path": "/learn/preserving-and-resetting-state"
         },
         {
-          "title": "Extracting State Logic into a Reducer",
+          "title": "Estrarre la logica dello State in un Reducer",
```

```suggestion
          "title": "Estrarre la Logica dello State in un Reducer",
```
Usiamo lo stesso stile della versione originale, per convenzione :)